### PR TITLE
None-ya Business

### PR DESCRIPTION
**Pull request by drewpearce**
I’m gonna run out of None jokes soon.
Lego checks for None also has error exception handling now.

### CODE REVIEW

File: legos/devopsy.py [hunk 1]
```diff
@@ -12,7 +12,13 @@
 class Devopsy(Lego):
 
     def listening_for(self, message):
-        return message['text'].split()[0] == '!devopsy'
+        if message['test'] is not None:
```
Comment (sacres) [inline legos/devopsy.py]: Thanks for the PR @drewpearce! Seems you have a typo here: 's/'test'/'text'/'

**Review by sacres — CHANGES_REQUESTED** [review]
Also will need to rebase due to previous merge.

File: legos/devopsy.py (hunk 1 above)
Comment (drewpearce) [reply legos/devopsy.py]: fixed typo

**Comment by sacres** [comment]
Thanks @drewpearce but you still need to rebase against master.

**Comment by bbriggs** [comment]
Looks good. We'll change the interpolation in another PR.

**Review by bbriggs — APPROVED** [review]

**Comment by sacres** [comment]
What? It reverts my changes... which are already merged in master..

**Comment by bbriggs** [comment]
That's not what I'm seeing in the diff.
http://imgur.com/a/pRzt5

What are you seeing?

**Comment by drewpearce** [comment]
closing to avoid this mess. will resubmit later off of an updated branch.

**Pull request closed without merging**